src/Rules/NoEnvCallsOutsideOfConfigRule.php

```diff
@@ -60,4 +64,9 @@ protected function isCalledOutsideOfConfig(FuncCall $call, Scope $scope): bool
     {
         return str_starts_with($scope->getFile(), config_path()) === false;
     }
+
+    protected function isCalledOutsideOfPackageConfig(FuncCall $call, Scope $scope): bool
```

Comment:
You do not need `$call`, the first parameter.

Reply:
I copied it one line above, I thought it was for a good reason, since it is not used there either.
I'll delete it then
https://github.com/larastan/larastan/blob/8d433ed5304abdc437439b5b59fa040d6cf43de2/src/Rules/NoEnvCallsOutsideOfConfigRule.php#L59-L62

Reply:
Fixed